packages/blockchain-wallet-v4-frontend/src/data/components/brokerage/sagas.ts

```diff
@@ -71,10 +71,10 @@ export default ({ api }: { api: APIType }) => {
       if (typeof account === 'string' && bankCredentials) {
         // Yapily
         const domainsR = yield select(selectors.core.walletOptions.getDomains)
-        const { yapilyCallbackUrl } = domainsR.getOrElse({
-          yapilyCallbackUrl: 'https://www.blockchain.com/brokerage-link-success'
+        const { comRoot } = domainsR.getOrElse({
+          comRoot: 'https://www.blockchain.com'
```

Comment:
(minor): maybe to move it to const and reuse later on line 309

Reply:
yeah we really need to add a constants file for a lot things. been on my list